query-engine/connectors/sql-query-connector/src/database/sqlite.rs

```diff
@@ -38,30 +38,13 @@ impl FromSource for Sqlite {
             .map_err(|sql_error| sql_error.into_connector_error(&connection_info))?;
 
         let file_path = params.file_path;
+        let url = database_str.split('?').next();
 
-        let url_with_db = {
-            let db_name = std::path::Path::new(&file_path)
-                .file_stem()
-                .and_then(|stem| stem.to_str())
-                .ok_or_else(|| invalid_file_path_error(&file_path, &connection_info))?
-                .to_owned();
+        if url.is_none() || std::path::Path::new(url.unwrap()).file_stem().is_none() {
+            return Err(invalid_file_path_error(&file_path, &connection_info));
+        }
 
-            let mut splitted = database_str.split('?');
-            let url = splitted.next().unwrap();
-            let params = splitted.next();
-
-            let mut params: Vec<&str> = match params {
-                Some(params) => params.split('&').collect(),
-                None => Vec::with_capacity(1),
-            };
-
-            let db_name_param = format!("db_name={}", db_name);
-            params.push(&db_name_param);
-
-            format!("{}?{}", url, params.join("&"))
```

Comment:
We no longer add `db_name` to the connection string. Quaint no longer supports it anyway since https://github.com/prisma/quaint/pull/252. Instead, we use the default db name called `main`.